yarn.lock

```diff
@@ -1388,6 +1388,11 @@ case@1.6.3, case@^1.6.3:
   resolved "https://registry.yarnpkg.com/case/-/case-1.6.3.tgz#0a4386e3e9825351ca2e6216c60467ff5f1ea1c9"
   integrity sha512-mzDSXIPaFwVDvZAHqZ9VlbyF4yyXRuX6IvB06WvPYkqJVO24kX1PPhv9bfpKNFZyxYFmmgo03HUiD8iklmJYRQ==
 
+cdk-nag@^2.12.13:
+  version "2.12.13"
+  resolved "https://amazon.goshawk.aws.a2z.com:443/npm/shared/cdk-nag/-/cdk-nag-2.12.13.tgz#b54fbde5d5370b45b02954b9616b0215cb176048"
```

Comment:
Make sure you're not referring to the internal registry.

Reply:
Thanks for pointing this out. I am still baffled how these registries are picked one over another :).